algorithms/ar-bsrh/javascript/bst.js

```diff
@@ -0,0 +1,152 @@
+class Node {
+    constructor(data) {
+        this.right = null;
+        this.left = null;
+        this.data = data;
+    }
+}
+
+class Bst {
+    constructor() {
+        this.root = null;
+    }
+
+    insert(data) {
```

Comment:
Codacy found an issue: [Method 'insert' has a complexity of 8. Maximum allowed is 4.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6264809)